driver-core/src/test/unit/com/mongodb/internal/connection/TestConnectionPoolListener.java

```diff
@@ -89,7 +89,7 @@ public TestConnectionPoolListener() {
     }
 
     private <T> boolean containsEvent(final Class<T> eventClass, final int expectedEventCount) {
-        return countEvents(eventClass) == expectedEventCount;
+        return countEvents(eventClass) >= expectedEventCount;      // TODO
```

Comment:
Whats the todo here?

Reply:
I think it's correct as is with the change, as you might have more events than expected, just not less.  Removed  the TODO